yarn-project/aztec/src/cli/cmds/start_node.ts

```diff
@@ -62,62 +56,41 @@ export async function startNode(
 
   userLog(`Initial funded accounts: ${initialFundedAccounts.map(a => a.toString()).join(', ')}`);
 
-  const { genesisArchiveRoot, prefilledPublicData, fundingNeeded } = await getGenesisValues(initialFundedAccounts);
+  const { genesisArchiveRoot, prefilledPublicData } = await getGenesisValues(initialFundedAccounts);
 
   userLog(`Genesis archive root: ${genesisArchiveRoot.toString()}`);
 
   const followsCanonicalRollup =
     typeof nodeConfig.rollupVersion !== 'number' || (nodeConfig.rollupVersion as unknown as string) === 'canonical';
 
-  // Deploy contracts if needed
-  if (nodeSpecificOptions.deployAztecContracts || nodeSpecificOptions.deployAztecContractsSalt) {
-    let account;
-    if (nodeSpecificOptions.publisherPrivateKey) {
-      account = privateKeyToAccount(nodeSpecificOptions.publisherPrivateKey);
-    } else if (options.l1Mnemonic) {
-      account = mnemonicToAccount(options.l1Mnemonic);
-    } else {
-      throw new Error('--node.publisherPrivateKey or --l1-mnemonic is required to deploy L1 contracts');
-    }
-    // REFACTOR: We should not be calling a method from sandbox on the prod start flow
-    await deployContractsToL1(nodeConfig, account!, undefined, {
-      assumeProvenThroughBlockNumber: nodeSpecificOptions.assumeProvenThroughBlockNumber,
-      salt: nodeSpecificOptions.deployAztecContractsSalt,
-      genesisArchiveRoot,
-      feeJuicePortalInitialBalance: fundingNeeded,
-    });
+  if (!nodeConfig.l1Contracts.registryAddress || nodeConfig.l1Contracts.registryAddress.isZero()) {
+    throw new Error('L1 registry address is required to start Aztec Node without --deploy-aztec-contracts option');
```

Comment:
I think this error message is not relevant anymore, should just be ''L1 registry address is required to start Aztec Node'